src/starknet-types.ts

```diff
@@ -102,6 +102,28 @@ export interface Transaction {
     transaction_index: number;
 }
 
+export interface FunctionInvocation {
+    call_type: string;
+    calldata: string[];
+    caller_address: string;
+    class_hash: string;
+    contract_address: string;
+    entry_point_type: string;
+    events: unknown[];
```

Comment:
How come it's using `unknown` here?

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/264/commits/a38b37733e7611035080a544963379eb4af3ce43